tests/Features/Contexts/FeatureContext.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2018 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Features\Contexts;
+
+use Behat\Gherkin\Node\TableNode;
+use Behat\MinkExtension\Context\MinkContext;
+use Behat\Symfony2Extension\Context\KernelDictionary;
+use Illuminate\Database\Capsule;
+use Illuminate\Database\Connection;
+use OpenCFP\Domain\Model\Talk;
+use OpenCFP\Domain\Model\User;
+use OpenCFP\Domain\Services\AccountManagement;
+use Phinx\Console\PhinxApplication;
+use Symfony\Component\Console\Input\StringInput;
+use Symfony\Component\Console\Output\NullOutput;
+
+final class FeatureContext extends MinkContext
```

Comment:
This is probably too much for this PR, but we should look into splitting up contexts where possible, so this class does not become too large to manage

Reply:
I would recommend `SpeakerContext` and `AdminContext` and call it at that. I would prefer to make that the work of another PR as the point of this one was to get more-or-less a baseline.